phpstan.neon

```diff
@@ -13,11 +13,8 @@ parameters:
         - lib/Test.php
         - lib/WorseReflection/Core/SourceCodeLocator/InternalStubs/*
         - lib/Extension/LanguageServerCodeTransform/Tests/Stub/*
-    reportUnmatchedIgnoredErrors: false
+    reportUnmatchedIgnoredErrors: true
```

Comment:
i set this to `false` because it happens on almost change which is rather unnecessary. It's easier to just regenerate the baseline every now and then. so would prefer to levae this as `false`

Reply:
Yeah, I have only enabled that because I wanted to see how many errors I can remove.

Reply:
I find it better to just `generate-baseline` and then do a diff to see how much was removed, as long as PHPStan is passing it's safe to do that.

Reply:
Yeah, I do the same. Just forgot to revert the change.